package/MDAnalysis/core/groups.py

```diff
@@ -665,7 +713,7 @@ class AtomGroup(object):
 
     """
 
-    level = 'atom'
+    order = 'group'
```

Comment:
I'm not sure order is right here.. but I can't really think of a better name.. `plurality` maybe?  Or could just have a boolean `_is_group`